packages/perseus/src/renderer-util.ts

```diff
@@ -66,14 +65,17 @@ export function scorePerseusItem(
     strings: PerseusStrings,
     locale: string,
 ): PerseusScore {
+    let convertedRenderData = perseusRenderData;
+    let convertedUserInputMap = userInputMap;
+
     // Check if the PerseusRenderer object contains any deprecated widgets that need to be converted
     const mustConvertData = conversionRequired(perseusRenderData);
-    const convertedRenderData = mustConvertData
-        ? convertDeprecatedWidgets(perseusRenderData) // Convert deprecated widgets to their modern equivalents
-        : perseusRenderData;
-    const convertedUserInputMap = mustConvertData
-        ? convertUserInputData(userInputMap) // Convert deprecated user input data keys to their modern equivalents
-        : userInputMap;
+    if (mustConvertData) {
+        const {convertedRubric, convertedUserData} =
+            convertDeprecatedWidgetsForScoring(perseusRenderData, userInputMap);
+        convertedRenderData = convertedRubric;
+        convertedUserInputMap = convertedUserData;
+    }
 
```

Comment:
I could do a ternary destruction here, but it looked really messy, imo. I preferred the simple if statement for simple legibility - but I'd be happy hear any alternative opinions/suggestions! 

```suggestion
    const {convertedRubric, convertedUserData} = conversionRequired(perseusRenderData)
        ? convertDeprecatedWidgetsForScoring(perseusRenderData, userInputMap)
        : {convertedRubric: perseusRenderData, convertedUserData: userInputMap};

    // There seems to be a chance that PerseusRenderer.widgets might include
    // widget data for widgets that are not in PerseusRenderer.content,
    // so this checks that the widgets are being used before scoring them
    const usedWidgetIds = getWidgetIdsFromContent(convertedRubric.content);
    const scores = scoreWidgetsFunctional(
        convertedRubric.widgets,
        usedWidgetIds,
        convertedUserData,
        strings,
        locale,
    );
```